code/modules/client/preferences_toggles.dm

```diff
@@ -29,6 +29,14 @@
 		prefs.toggles ^= TOGGLE_FULLSCREEN
 		toggle_fullscreeny(prefs.toggles & TOGGLE_FULLSCREEN)
 
+/client/verb/toggle_screenshake()
+	set name = "Toggle Screen Shake"
+	set category = "Options"
+	set desc = ""
+	if(prefs)
+		prefs.shake = !prefs.shake
+		to_chat(src, "Screen shake toggled")
```

Comment:
```suggestion
		if(prefs.shake)
			to_chat(src, "Screen shake enabled")
		else
			to_chat(scr, "Screen shake disabled")
```
Please test this if you do commit it directly from github!

Reply:
Appreciated, I didn't really know how code that.